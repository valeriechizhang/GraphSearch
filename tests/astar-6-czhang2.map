1   1   1   1   1   1   1   1
1   1   1   1   1   1   1   1
1   1   1   1   1   1   1   1
1   1   1   1   1   1   1   1
1   1   1   1   1   1   1   1
1   50  100 150 200 250 300 1