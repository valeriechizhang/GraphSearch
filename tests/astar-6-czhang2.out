Solution steps: ['N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'S']
Solution cost: 9
Number of states considered: 10

Frontier:
Pos=(2, 0) Moves=['N', 'E', 'E', 'S'] Cost=9805
Pos=(1, 0) Moves=['E'] Cost=2402
Pos=(7, 2) Moves=['N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N'] Cost=9
Pos=(6, 2) Moves=['N', 'E', 'E', 'E', 'E', 'E', 'E', 'N'] Cost=8
Pos=(5, 2) Moves=['N', 'E', 'E', 'E', 'E', 'E', 'N'] Cost=7
Pos=(4, 2) Moves=['N', 'E', 'E', 'E', 'E', 'N'] Cost=6
Pos=(3, 2) Moves=['N', 'E', 'E', 'E', 'N'] Cost=5
Pos=(2, 2) Moves=['N', 'E', 'E', 'N'] Cost=4
Pos=(1, 2) Moves=['N', 'E', 'N'] Cost=3
Pos=(0, 2) Moves=['N', 'N'] Cost=2

Closed List:
Pos=(0, 0) Moves=[] Cost=0
Pos=(0, 1) Moves=['N'] Cost=1
Pos=(1, 1) Moves=['N', 'E'] Cost=2
Pos=(2, 1) Moves=['N', 'E', 'E'] Cost=3
Pos=(3, 1) Moves=['N', 'E', 'E', 'E'] Cost=4
Pos=(4, 1) Moves=['N', 'E', 'E', 'E', 'E'] Cost=5
Pos=(5, 1) Moves=['N', 'E', 'E', 'E', 'E', 'E'] Cost=6
Pos=(6, 1) Moves=['N', 'E', 'E', 'E', 'E', 'E', 'E'] Cost=7
Pos=(7, 1) Moves=['N', 'E', 'E', 'E', 'E', 'E', 'E', 'E'] Cost=8
Pos=(7, 0) Moves=['N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'S'] Cost=9
